packages/fast-components-react-base/src/breadcrumb/examples.data.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from "react";
+import { ComponentFactoryExample } from "@microsoft/fast-development-site-react";
+import schema from "./breadcrumb.schema.json";
+import Breadcrumb, { BreadcrumbManagedClasses, BreadcrumbProps } from "./breadcrumb";
+import Documentation from "./.tmp/documentation";
+import reactHTMLElementSchema from "../../app/components/react-html-element.schema.json";
+
+const managedClasses: BreadcrumbManagedClasses = {
+    managedClasses: {
+        breadcrumb: "breadcrumb",
+        breadcrumb_item: "breadcrumb-item",
+        breadcrumb_item__current: "breadcrumb-item-current",
+        breadcrumb_ol: "breadcrumb-ol",
+        breadcrumb_separator: "breadcrumb-separator",
+    },
+};
+
+const examples: ComponentFactoryExample<BreadcrumbProps> = {
+    name: "Breadcrumb",
+    component: Breadcrumb,
+    schema: schema as any,
+    documentation: <Documentation />,
+    detailData: {
+        ...managedClasses,
+        label: "breadcrumb",

```

Comment:
Since label doesn't change at all in the examples, I would recommend setting this at the top of the file as a constant and passing it using the spread operator.